README.md

```diff
@@ -1,2 +1,150 @@
 # SuanShu
-This extension fixes testcases and serialized the library.
\ No newline at end of file
+
+This extension fixes testcases and serialized the library.
+
+## Installation
+
+The Maven repository for this project can be found at https://mvnrepository.com/artifact/io.deephaven/SuanShu
+
+### pom.xml
+
+```
+<!-- https://mvnrepository.com/artifact/io.deephaven/SuanShu -->
+<dependency>
+    <groupId>io.deephaven</groupId>
+    <artifactId>SuanShu</artifactId>
+    <version>0.1.0</version>
+</dependency>
+```
+
+### Gradle
+
+```
+// https://mvnrepository.com/artifact/io.deephaven/SuanShu
+implementation group: 'io.deephaven', name: 'SuanShu', version: '0.1.0'
```

Comment:
We should prefer the shorter form: `implementation 'io.deephaven:SuanShu:0.1.0'`